web/.storybook/main.js

```diff
@@ -49,6 +53,19 @@ module.exports = {
       ...storybookConfig.resolve,
       ...configFactory.createDefaultConfig().resolve,
     };
+    storybookConfig.resolve.alias['access-graph'] = path.join(
+      __dirname,
+      'mocks',
+      'access-graph-mock.jsx'
+    );
+    if (!enterpriseTeleportExists) {
+      delete storybookConfig.resolve.alias['e-teleport'];
+      storybookConfig.resolve.alias['e-teleterm'] = path.join(
```

Comment:
It may not be clear why`e-teleterm` has a special treatment here (comparing to `e-teleport`). I know that we have a link to the GH issue in the `DocumentAccessRequests.tsx`, but I'd repeat it here.